build/hellobuild.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Hello Build
+description: Build Hello World
+keywords: build, buildx, buildkit, getting started, Dockerfile, image layers, build instructions, build context
+---
+
+## Hello Build!
+
+It all starts with a Dockerfile.
+
+Dockerfiles are text files containing instructions. Dockerfiles adhere to a specific format and contain a **set of instructions** for which you can find a full reference in the [Dockerfile reference](https://docs.docker.com/engine/reference/builder/).
```

Comment:
Could you use relative paths for links within the docs?

Reply:
Sure, will do.